app/lib/common/widgets/redact_content.dart

```diff
@@ -89,13 +89,7 @@ class RedactContentWidget extends ConsumerWidget {
   }
 
   void redactContent(BuildContext ctx, WidgetRef ref, String reason) async {
-    showAdaptiveDialog(
-      context: (ctx),
-      builder: (ctx) => DefaultDialog(
-        title: Text(L10n.of(ctx).removingContent),
-        isLoader: true,
-      ),
-    );
+    EasyLoading.show(status: L10n.of(ctx).removingContent, dismissOnTap: false);
```

Comment:
can you explain to me why we do not want to dismiss on Tap? I've seen that a few times now, but I don't think we always want to be that much into the users face ... if they want the loading to disappear on tap, we should allow for that, no?

Reply:
Fixed by https://github.com/acterglobal/a3/pull/1580/commits/8e8f1e5068f1c5fdfc823cc91d330eac2edeee76